core/sdk-core/src/main/java/software/amazon/awssdk/core/internal/http/pipeline/stages/SigningStage.java

```diff
@@ -132,6 +141,8 @@ private SdkHttpFullRequest signRequest(SdkHttpFullRequest request, RequestExecut
 
             SdkHttpFullRequest signedRequest = measuredSign.left();
 
+            // TODO: This case does not apply to SigningStage as event stream operations are not supported by SyncClients that
```

Comment:
Why don't we remove it?

Reply:
I was leaving old code as-is for sometime, until more testing is done. I might remove it before release.

Reply:
Are there any particular test cases that would be written later that we wouldn't remember to write if this were to be removed? I'd think that the tests would show if this piece of code was missing. It seems there are a lot of TODOs still to resolve before release... but if we're planning an activity to just go through all those and resolve them after general code complete, that should work too.

Reply:
Yeah, going through all TODOs is in my delivery plan for after general code complete.